persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml

```diff
@@ -149,7 +159,7 @@
     </select>
 
     <select id="getSampleInternalIds" resultType="Integer">
-        SELECT sample.INTERNAL_ID
+        SELECT DISTINCT sample.INTERNAL_ID
```

Comment:
Why did you need to add DISTINCT? I looked briefly, but I couldn't figure out how you were seeing duplicates.